### PR TITLE
Color setup and logo setup

### PR DESCRIPTION
**Pull request by MaximilienG1**
I've set up the exact colors we'll need to use in the application, i.e. the colors for the background, the buttons and the text in these different colors.  
I've also set up the svg logos for the different pages of our application, for greater clarity.

### CODE REVIEW

File: settings.gradle.kts
Comment (JulesHerrscher) [inline settings.gradle.kts:1]: The project name is still Assos in my project, we should pay attention to this change

File: app/src/main/java/com/swent/assos/ui/screens/Profil.kt
Comment (JulesHerrscher) [inline app/src/main/java/com/swent/assos/ui/screens/Profil.kt:1]: the expermental3Api seems to not be mandatory for such a small composable.

**Review by JulesHerrscher — CHANGES_REQUESTED** [review]
Good pull request, it is needed to discuss about the name of the application in the settings file

**Review by Reza-rgb — CHANGES_REQUESTED** [review]
It will be very handy to have all these pretty colours, thanks ! The different logos look good also ! I just highlighted a few minor adjustment about the general organisation. As Jules said, you also need to format the new code so that you pass the CI and we can merge with the main branch

File: settings.gradle.kts
Comment (Tikipiou) [reply settings.gradle.kts:1]: It should not be a problem

**Pull request closed without merging**